subsys/modem/backends/modem_backend_uart_async.c

```diff
@@ -153,12 +144,15 @@ static int modem_backend_uart_async_open(void *data)
 	 * used to store received data.
 	 */
 	ret = uart_rx_enable(backend->uart, backend->async.receive_bufs[0],
-			     backend->async.receive_buf_size, 3000);
+			     backend->async.receive_buf_size, 30000);
```

Comment:
Should there be a define or kconfig for this value?

Reply:
Added a Kconfig for this value